pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -1032,9 +1030,10 @@ protected void startLeaderElectionService() {
                             if (loadResourceQuotaTask != null) {
                                 loadResourceQuotaTask.cancel(false);
                             }
-                            loadSheddingTask = loadManagerExecutor.scheduleAtFixedRate(
-                                    new LoadSheddingTask(loadManager),
-                                    loadSheddingInterval, loadSheddingInterval, TimeUnit.MILLISECONDS);
+                            loadSheddingTask = loadManagerExecutor.schedule(
```

Comment:
It seems not right to cancel `loadSheddingTask` like this.

Reply:
like this:
<img width="936" alt="image" src="https://user-images.githubusercontent.com/19296967/179516593-ea1925ae-852a-4f72-99fd-e0b821c578dd.png">

Reply:
fixed! PTAL,thanks! @Jason918